app/src/main/java/com/main/mealplanner/UI/MainFragment.kt

```diff
@@ -18,35 +19,37 @@ import kotlinx.android.synthetic.main.main_fragment.*
 import java.util.ArrayList
 
 class MainFragment: Fragment(){
-    companion object {
-        fun newInstance() = MainFragment()
-    }
 
     private lateinit var viewModel: MainViewModel
     private var _events = ArrayList<UsageEvents.Event>()
+
     override fun onCreateView(
         inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View {
         return inflater.inflate(R.layout.main_fragment, container, false)
     }
+
     override fun onActivityCreated(savedInstanceState: Bundle?) {
```

Comment:
I extracted the behavior that observes the live data into its own method, and out of the onCreate activity event handler. If this code needs to grow in the future, having it in its own method will be helpful.